content/id/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -0,0 +1,371 @@
+---
+title: Alokasi Sumber Daya Dinamis
+content_type: concept
+weight: 65
+api_metadata:
+- apiVersion: "resource.k8s.io/v1alpha3"
+  kind: "DeviceTaintRule"
+- apiVersion: "resource.k8s.io/v1beta1"
+  kind: "ResourceClaim"
+- apiVersion: "resource.k8s.io/v1beta1"
+  kind: "ResourceClaimTemplate"
+- apiVersion: "resource.k8s.io/v1beta1"
+  kind: "DeviceClass"
+- apiVersion: "resource.k8s.io/v1beta1"
+  kind: "ResourceSlice"
+- apiVersion: "resource.k8s.io/v1beta2"
+  kind: "ResourceClaim"
+- apiVersion: "resource.k8s.io/v1beta2"
+  kind: "ResourceClaimTemplate"
+- apiVersion: "resource.k8s.io/v1beta2"
+  kind: "DeviceClass"
+- apiVersion: "resource.k8s.io/v1beta2"
+  kind: "ResourceSlice"
+---
+
+<!-- overview -->
+
+{{< feature-state feature_gate_name="DynamicResourceAllocation" >}}
+
+Alokasi sumber daya dinamis (_dynamic resource allocation_) adalah API untuk meminta dan berbagi sumber daya antara Pod dan kontainer di dalam sebuah Pod. Ini merupakan generalisasi dari API PersistentVolume untuk sumber daya generik. Biasanya, sumber daya tersebut bisa berupa perangkat seperti GPU.
+
+_Driver_ sumber daya pihak ketiga bertanggung jawab untuk melacak dan mempersiapkan sumber daya, dengan alokasi sumber daya yang ditangani oleh Kubernetes melalui parameter terstruktur (diperkenalkan pada Kubernetes 1.30). Berbagai jenis sumber daya mendukung parameter sembarang untuk mendefinisikan kebutuhan dan inisialisasi.
+
+Kubernetes versi 1.26 hingga 1.31 menyertakan implementasi (alpha) dari DRA klasik, yang sekarang sudah tidak didukung lagi. Dokumentasi ini, yang ditujukan untuk Kubernetes versi {{< skew currentVersion >}}, menjelaskan pendekatan terkini untuk alokasi sumber daya dinamis dalam Kubernetes.
```

Comment:
I think we need to mention once for the abbreviation of DRA. 

Example: DRA (Dynamic Resource Allocation)